fvm/fvm_bench_test.go

```diff
@@ -399,91 +419,112 @@ func BenchmarkRuntimeTransaction(b *testing.B) {
+		benchTransaction(b, templateTx(100, `getAccount(signer.address).availableBalance`), false)
 	})
 	b.Run("get account and get storage used", func(b *testing.B) {
-		benchTransaction(b, templateTx(100, `getAccount(signer.address).storageUsed`))
+		benchTransaction(b, templateTx(100, `getAccount(signer.address).storageUsed`), false)
 	})
 	b.Run("get account and get storage capacity", func(b *testing.B) {
-		benchTransaction(b, templateTx(100, `getAccount(signer.address).storageCapacity`))
+		benchTransaction(b, templateTx(100, `getAccount(signer.address).storageCapacity`), false)
 	})
 	b.Run("get signer vault", func(b *testing.B) {
-		benchTransaction(b, templateTx(100, `let vaultRef = signer.borrow<&FlowToken.Vault>(from: /storage/flowTokenVault)!`))
+		benchTransaction(
+			b,
+			templateTx(100, `let vaultRef = signer.borrow<&FlowToken.Vault>(from: /storage/flowTokenVault)!`),
+			false)
 	})
 	b.Run("get signer receiver", func(b *testing.B) {
-		benchTransaction(b, templateTx(100, `let receiverRef =  getAccount(signer.address)
+		benchTransaction(
+			b,
+			templateTx(100, `let receiverRef =  getAccount(signer.address)
 				.getCapability(/public/flowTokenReceiver)
-				.borrow<&{FungibleToken.Receiver}>()!`))
+				.borrow<&{FungibleToken.Receiver}>()!`),
+			false)
 	})
 	b.Run("transfer tokens", func(b *testing.B) {
-		benchTransaction(b, templateTx(100, `
-			let receiverRef =  getAccount(signer.address)
-				.getCapability(/public/flowTokenReceiver)
-				.borrow<&{FungibleToken.Receiver}>()!
-			
-			let vaultRef = signer.borrow<&FlowToken.Vault>(from: /storage/flowTokenVault)!
-
-			receiverRef.deposit(from: <-vaultRef.withdraw(amount: 0.00001))
-			`))
+		benchTransaction(
+			b,
+			templateTx(100, `
+                let receiverRef =  getAccount(signer.address)
+                    .getCapability(/public/flowTokenReceiver)
+                    .borrow<&{FungibleToken.Receiver}>()!
+
+                let vaultRef = signer.borrow<&FlowToken.Vault>(from: /storage/flowTokenVault)!
+
+                receiverRef.deposit(from: <-vaultRef.withdraw(amount: 0.00001))
+			`),
+			false)
 	})
 	b.Run("load and save empty string on signers address", func(b *testing.B) {
-		benchTransaction(b, templateTx(100, `
+		benchTransaction(
+			b,
+			templateTx(100, `
 				signer.load<String>(from: /storage/testpath)
 				signer.save("", to: /storage/testpath)
-			`))
+			`),
+			false)
 	})
 	b.Run("load and save long string on signers address", func(b *testing.B) {
-		benchTransaction(b, templateTx(100, fmt.Sprintf(`
+		benchTransaction(
+			b,
+			templateTx(100, fmt.Sprintf(`
 				signer.load<String>(from: /storage/testpath)
 				signer.save("%s", to: /storage/testpath)
-			`, longString)))
+			`, longString)),
+			false)
 	})
 	b.Run("create new account", func(b *testing.B) {
-		benchTransaction(b, templateTx(50, `let acct = AuthAccount(payer: signer)`))
+		benchTransaction(b, templateTx(50, `let acct = AuthAccount(payer: signer)`), true)
```

Comment:
Note: this bench test still uses service account as payer, hence interaction is "zero"